### ISSUE TITLE
'n.d.' for undated publications is not disambiguated

### ISSUE THREAD

**Issue by spwhitton** [jgm/pandoc-citeproc#325]
Hello,

Suppose that I have two citation keys, `@dobbsmsa` and `@dobbsmsb`.  They are two unpublished manuscripts by the same author, with different titles.  Now suppose I want to cite both:

    In recent essays Dobbs [-@dobbsmsa, p. 1; -@dobbsmsb, pp. 2--3] ...

This will produce "In recent essays Dobbs (n.d., 1, 2--3)" which suggests I'm referring to two pages from the same work.  What I want is something like this: "Dobbs (n.d. a, 1, n.d. b, 2--3)".

I'm not sure if this issue is actually in pandoc-citeproc or somewhere else in the toolchain, so please excuse me if I'm filing in the wrong place.

**Comment by jgm** [comment]
You might try a few different styles to see if this is a
style-dependent issue or a pandoc-citeproc one.

**Comment by spwhitton** [comment]
Hello,

On Fri, Mar 09 2018, John MacFarlane wrote:

> You might try a few different styles to see if this is a
> style-dependent issue or a pandoc-citeproc one.

From the upstream styles repo, british-journal-of-political-science.csl
and chicago-author-date.csl show the bug, while university-of-york-chicago.csl
does not.

I don't think this really reveals whether or not it is a pandoc-citeproc
bug, because it is possible that the bug is triggered by certain styles,
or that it is present in both british-journal-of-political-science.csl
and in chicago-author-date.csl.

-- 
Sean Whitton

**Comment by jgm** [comment]
I wasn't able to reproduce this in chicago-author-date.csl (pandoc-citeproc's built-in default).
Can you post a minimal example that suffices to reproduce it?

**Comment by jgm** [comment]
Okay, I was able to reproduce.
```
% pandoc 325.md -F pandoc-citeproc -t plain
(Wright, n.d.; Wright, n.d.)

Wright, Sewall. n.d. “Title.”

———. n.d. “Title2.”
```
where 325.md is
```
---
references:
- id: my2
  type: manuscript
  author:
  - family: Wright
    given: Sewall
  title: Title
  page: 1-2

- id: my1
  type: manuscript
  author:
  - family: Wright
    given: Sewall
  title: Title2
  page: 395-420
...

[@my1; @my2]
```

**Comment by spwhitton** [comment]
Hello,

On Sun 14 Oct 2018 at 03:16PM -0700, John MacFarlane wrote:

> Closed #325 via e3bfecd95bf34cb95226270e1bc995e7fd884824.

Thank you!

-- 
Sean Whitton
